app/assets/javascripts/initializers/initializeReadingListIcons.js

```diff
@@ -102,6 +102,7 @@ function buttonFormData(button) {
   var formData = new FormData();
   formData.append('reactable_type', 'Article');
   formData.append('reactable_id', button.dataset.reactableId);
+  formData.append('collection', button.dataset.collection);
```

Comment:
This was a great discovery to use the formData() to pass the current selected collection title to the BE. Perhaps we should have named it something more descriptive like "collectionDropboxValue"?